### ISSUE TITLE
Documentation: Review docs in preparation for engagement with other institutions

### ISSUE THREAD

**Issue by bbranan** [eclipse-pass/main#539]
Review the documentation in: https://github.com/eclipse-pass/main/tree/main/docs. Consider the following questions:
* Is the purpose of the PASS project clear?
* Is the current organization/structure of the documentation easy to understand and navigate?
* Are there gaps in the content that would be expected by newcomers?
* Is the method of getting started with the project clear?
* Is it clear how to contribute to the project?
* Is there documentation that is out of date and should be removed?

**Expected Outcome: Set of recommendations for improvement of the PASS documentation.**

Note: The work of this ticket is to perform the analysis needed to develop recommendations for improvement. Another ticket will follow to make the necessary updates.

**Comment by tsande16** [comment]
@cshannon2 I spoke with Bill and he asked for me to share these thoughts on this overarching documentation ticket. I am happy to help and collaborate on a user-friendly step by step installation guide for ourselves and external collaborators:

Having navigated the task of rediscovering the nihms-loader process due to lost knowledge transfer, it occurred to me that creating a straightforward, step-by-step installation/setup manual for PASS would greatly benefit not only our team but also external collaborators and other institutions. When I initiated the installation for our DSpace playground, I found these guides to be decent, so something akin to this: [DSpace Guide](https://wiki.lyrasis.org/display/DSDOC7x/Installing+DSpace) + [Introduction](https://wiki.lyrasis.org/display/DSDOC7x/Introduction). (Although I am totally open to other organization and  dissemination methods as well.)

While we already have decent documentation for developers across our Google Drive and READMEs, I believe a systematic guide would ease the process, especially considering the complex dependencies inherent in our data loader. It would help alleviate confusion caused by outdated information within our current documents.

**Comment by tsande16** [comment]
Adding some bullet points from our retreat that I took. @cshannon2, please let me know if you have anything else to add or if you have notes that are different from this. 

- Recommendation about a system to use for on-boarding another institution.
- Recommendation on on-boarding documentation.

**Comment by dkriethof** [comment]
Per our retreat, keep in mind:

- What pieces need to get PASS in place
- What are those pieces?
- Make decision for system we should use to manage all of this

**Comment by tsande16** [comment]
Next steps after #618 

Need to define a set of selection criteria that is most important to our team/community: #843

**Comment by tsande16** [comment]
List of recommendations for PASS documentation:

https://docs.google.com/document/d/1Ac8q3xVz6fMWulJZAqKbA6dblce5qfPwxqdbzgU3aiM/edit?usp=sharing
https://docs.google.com/document/d/1g_boyxAtF3zxAtEdpfGcf-tURK81eBmDlqFQUkQMdg4/edit?usp=drive_link
https://docs.google.com/presentation/d/1zb6J2hf6DfXqf9Wuu_F229OawUyYeGE0/edit?usp=drive_link&ouid=102870018102164937184&rtpof=true&sd=true
https://docs.google.com/document/d/1oMNZyz6Ub4IMm4QtfUVl08lQ0pmzoxU9HTLuS8HF-v0/edit?usp=drive_link
https://docs.google.com/spreadsheets/d/1ejovIlU5MnLGcKEwvdjW_7Pvy3Blvg1erjgMuOQzeMI/edit?usp=drive_link

In addition there have been several meetings and discussions surrounding documentation and we have a good idea of what to do for our next steps. After discussing with @dkriethof and @cshannon2 we decided this ticket can be be closed with the research that has been done.
